omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -171,14 +171,20 @@ def ldap_authentication_enabled?
       !(node['private_chef']['ldap'].nil? || node['private_chef']['ldap'].empty?)
   end
 
-  def repository_configured?(name)
+  def platform_package_suffix
     case node['platform_family']
-    when 'rhel'
-      File.exists? "/etc/yum.repos.d/#{name}.repo"
     when 'debian'
-      File.exists? "/etc/apt/sources.list.d/#{name}.list"
+      'deb'
+    when 'rhel', 'suse'
+      'rpm'
     else
-     false
+      # TODO: probably don't actually want to fail out?
+      raise "I don't know how to install addons for platform family: #{node['platform_family']}"
```

Comment:
Maybe we should provide a flag to this command to allow the user to override and specify which repo type they want, and if they hit this, we could tell them about the flag?